### PR TITLE
Feature react2

### PR DESCRIPTION
**Pull request by Aravindkaspe**
Please Review code as I have made fully functional Login Page and Quote Form

### CODE REVIEW

File: Source/Package.json
Comment (chiranjeevigundu) [inline Source/Package.json:1]: Dependencies on Package.json looks good and works fine with upcoming merges.

File: Source/babel.config.js
Comment (chiranjeevigundu) [inline Source/babel.config.js:1]: bable configuration is good

File: Source/backend/index.js [hunk 1]
```diff
@@ -2,44 +2,49 @@ import express from "express";
...
+  optionsSuccessStatus: 204
+}));
+
 app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
-app.use(
-	cors({
-		"Access-Control-Allow-Origin": "*",
-		methods: "GET,HEAD,PUT,PATCH,POST,DELETE",
```
Comment (chiranjeevigundu) [inline Source/backend/index.js:15]: Seems these API calls were being handled somewhere else.

**Comment by kishore7860** [comment]
The CSS is well-structured and clear; however, consider using shorthand properties for margin and padding. Additionally, to enhance accessibility, include :focus styles for the button.

File: Source/backend/models/Quote.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		type: String,
+		required: true,
+		trim: true,
+	},
+	email: {
+		type: String,
+		required: true,
+		trim: true,
+		unique: true,
+		validate(value) {
```
Comment (chiranjeevigundu) [inline Source/backend/models/Quote.js:15]: Using email validator within the model is a good idea

File: Source/frontend/my-app/src/ContactForm.test.js
Comment (chiranjeevigundu) [inline Source/frontend/my-app/src/ContactForm.test.js:1]: This test file might belong to a different directory

**Review by chiranjeevigundu — APPROVED** [review]
Reviewed all frontend and Backend files, all functions are well designed and utilized at all stages. Excluded reviewing system default files in node_module directory.

**Comment by Amruth-Reddy-N** [comment]
Code works fine

**Comment by Amruth-Reddy-N** [comment]
Every thing is fine

**Pull request merged**